benchmarks/overrides_benchmark/bench.py

```diff
@@ -0,0 +1,67 @@
+import torch
+import time
+import argparse
+
+from common import SubTensor, WithTorchFunction, SubWithTorchFunction
+
+NUM_REPEATS = 1000
+NUM_REPEAT_OF_REPEATS = 1000
+
+
+def bench(t1, t2):
+    bench_times = []
+    for _ in range(NUM_REPEAT_OF_REPEATS):
+        time_start = time.time()
+        for _ in range(NUM_REPEATS):
+            torch.add(t1, t2)
+        bench_times.append(time.time() - time_start)
+
+    bench_time = float(torch.min(torch.Tensor(bench_times))) / 1000
+    bench_std = float(torch.std(torch.Tensor(bench_times))) / 1000
+
+    return bench_time, bench_std
+
+
+def main():
+    global NUM_REPEATS
+    global NUM_REPEAT_OF_REPEATS
+
+    parser = argparse.ArgumentParser(
+        description="Run the __torch_function__ benchmarks."
+    )
+    parser.add_argument(
+        "--nreps",
+        "-n",
+        type=int,
+        default=NUM_REPEATS,
+        help="The number of repeats for one measurement.",
+    )
+    parser.add_argument(
+        "--nrepreps",
+        "-m",
+        type=int,
+        default=NUM_REPEAT_OF_REPEATS,
+        help="The number of measurements.",
+    )
+    args = parser.parse_args()
+
+    NUM_REPEATS = args.nreps
+    NUM_REPEAT_OF_REPEATS = args.nrepreps
+
+    types = torch.Tensor, SubTensor, WithTorchFunction, SubWithTorchFunction
+
+    for t in types:
+        tensor_1 = t(1)
+        tensor_2 = t(2)
+
+        bench_min, bench_std = bench(tensor_1, tensor_2)
+        print(
+            "Type {0} had a minimum time of {1} \u03bcs"
+            " and a standard deviation of {2} \u03bcs.".format(
```

Comment:
You haven't fixed the issue that caused CI to fail?

```
Mar 26 20:13:22 + python bench.py -n 1 -m 1
Mar 26 20:13:22 ~/workspace/benchmarks/overrides_benchmark ~/workspace
Mar 26 20:13:22 Traceback (most recent call last):
Mar 26 20:13:22   File "bench.py", line 67, in <module>
Mar 26 20:13:22     main()
Mar 26 20:13:22   File "bench.py", line 61, in main
Mar 26 20:13:22     t.__name__, (10 ** 6) * bench_min, (10 ** 6) * bench_std,
Mar 26 20:13:22 UnicodeEncodeError: 'ascii' codec can't encode character '\u03bc' in position 54: ordinal not in range(128)
```

You may have to push to pytorch/pytorch on a branch named `ci-all/your-branch-name` to trigger the relevant ci job

Reply:
Ah, I assumed it was hitting that error when parsing the file, not when printing the char.

Reply:
> You may have to push to pytorch/pytorch on a branch named ci-all/your-branch-name to trigger the relevant ci job

Pushed to `ci-all/torch-function-benchmark`